python/ray/runtime_context.py

```diff
@@ -148,6 +240,15 @@ def current_placement_group_id(self):
         """
         return self.worker.placement_group_id
 
+    def get_current_placement_group_id(self) -> Optional[str]:
```

Comment:
Unless you feel strongly against it, could we rename to `get_placement_group_id`? The old apis will be removed eventually. It's better to be consistent with other new apis I think.